ynr/settings/deploy.py

```diff
@@ -36,6 +36,10 @@
 ADMINS = [("Dummy Admin", "dummy@example.com")]
 
 
+# Always run scheduled jobs in deployed environments
+ENABLE_SCHEDULED_JOBS = True
```

Comment:
I added this `ENABLE_SCHEDULED_JOBS` setting here. My thinking here is that most of the time you _don't_ want your dev copy to be running the same scheduled jobs that production runs, but it is useful to be able to enable that behaviour.
We could, of course, make this configurable task-by-task if we wanted.
The other thing I've done to start with is I've said this is off by default but on for all deployed environments.
We could make this an env var and then that would allow us to turn scheduled jobs on for prod but off in dev/staging. That might be useful